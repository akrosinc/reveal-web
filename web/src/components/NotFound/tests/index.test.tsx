import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import NotFound from '..';

describe('src/componenets/NotFound', () => {
  it('renderes NotFound correctly', () => {
    const wrapper = mount(<NotFound />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
