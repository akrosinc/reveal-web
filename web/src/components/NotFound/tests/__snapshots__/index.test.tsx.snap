// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/componenets/NotFound renderes NotFound correctly 1`] = `
<NotFound>
  <div>
    404!
  </div>
</NotFound>
`;
